osu.Game/Screens/Play/EpilepsyWarning.cs

```diff
@@ -24,7 +24,19 @@ public EpilepsyWarning()
             Alpha = 0f;
         }
 
-        public BackgroundScreenBeatmap DimmableBackground { get; set; }
+        private BackgroundScreenBeatmap dimmableBackground;
+
+        public BackgroundScreenBeatmap DimmableBackground
+        {
+            get => dimmableBackground;
+            set
+            {
+                dimmableBackground = value;
+
+                if (IsLoaded)
+                    updateBackgroundFade();
```

Comment:
Shouldn't this be checking the background's loaded state, or just using `ApplyToBackground()` itself? Not sure what the guarantees of the state of this background are at the time of set.

Reply:
This is to avoid `updateBackgroundFade` getting called when the `DimmableBackground` is set, which may be at a different point to when we want to display the fade (at `PopIn`).

Reply:
Seems fair enough now that I look at it again. Let's just go with this for now.